src/components/publishing/image_collection_item.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react"
+import styled from "styled-components"
+
+interface FillwidthItemProps extends React.HTMLProps<HTMLDivElement> {
+  width?: number
+  height?: number
+  margin?: number
+  key: string
+}
+
+const FillwidthItem: React.SFC<FillwidthItemProps> = props => {
+  const { key, ...rest } = Object.assign({}, props)
+  return <div {...rest} key={key} />
```

Comment:
Ah yes sorry I forgot to clean this up. I remember this was a sad attempt for me to figure out why I kept getting errors like:
```
Warning: Unknown prop `margin` on <div> tag. Remove this prop from the element.
```
even though I defined it in my prop definition.

What I didn't realize is that `<div>` itself isn't going to get the definition but props will be passed to it, so the way to write this instead is to not define a React.SFC at all, and just pass the proptypes to the styled component:
```
export default styled.div`       // previously styled(FillwidthItem)
 margin-right: ${(props: FillwidthItemProps) => props.margin + "px" || 0};
 width: ${(props: FillwidthItemProps) => props.width + "px" || "auto"};
 height: ${(props: FillwidthItemProps) => props.height + "px" || "auto"};
`
```
Does that make sense? Still kinda figuring out this stuff but this seems to work nicely.